README.md

```diff
@@ -57,8 +57,9 @@ Sofern dann an eine aufgerufenen URL **?pdf=1** angehängt wird, wird der Inhalt
 	      $dompdf = new Dompdf\Dompdf($options);
 	      $dompdf->set_option('defaultFont', 'Helvetica');
 	      $dompdf->set_option('dpi', '100');
-	      $dompdf->setPaper('A4', 'portrait');
-	      
+		  $dompdf->set_option('font_cache', rex_path::addonCache('pdfout', 'fonts'));
```

Comment:
indents ggf noch fixen? :)

Reply:
in meinem Editor sah es richtig aus, aber ja.

Reply:
Vermutlich spaces vs tabs

Reply:
```suggestion
	      $dompdf->set_option('font_cache', rex_path::addonCache('pdfout', 'fonts'));
```